HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+
+0.13.1
```

Comment:
These changes should be under `0.14.0`. Also please add line item(s) for the API/sdk refresh (API version being pointed at is a user facing change) and the to be done addition of telemetry options for `az dt twin telemetry send`.

Reply:
done